src/views/InfosDgv.vue

```diff
@@ -1,23 +1,25 @@
 <template>
   <div>
     <h2 v-if="display" class="fr-pl-2w fr-py-2w fr-m-0 infosdgv">
```

Comment:
```suggestion
    <h2 v-if="display" class="fr-pl-2w fr-py-1v fr-py-m-2w fr-m-0 infosdgv">
```

I think we can remove these paddings on small screens too. What do you think ?